src/main/java/com/faforever/client/main/LinksAndHelpController.java

```diff
@@ -0,0 +1,43 @@
+package com.faforever.client.main;
+
+import com.faforever.client.config.ClientProperties;
+import com.faforever.client.fx.AbstractViewController;
+import com.faforever.client.i18n.I18n;
+import com.faforever.client.theme.UiService;
+import javafx.scene.Node;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.layout.VBox;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
```

Comment:
Why is this prototype scope? It never changes.

Reply:
Controllers always have SCOPE_PROTOTYPE